t/db_dependent/api/v1/holdings.t

```diff
@@ -45,9 +46,48 @@ my $t = Test::Mojo->new('Koha::REST::V1');
 my $tx;
 
 subtest "list() tests" => sub {
-    plan skip_all => 'not implemented';
+    plan tests => 12;
 
     $schema->storage->txn_begin;
+
+    my ($patron, $session_patron) = create_user_and_session( { authorized => 0 } );
+
+    Koha::Holdings->search->delete;
+
+    $tx = $t->ua->build_tx( GET => '/api/v1/holdings' );
+    $tx->req->cookies( { name => 'CGISESSID', value => $session_patron } );
+    $t->request_ok( $tx )->status_is( 200 )
+        ->json_is( [] );
+
+    my $biblionumber = create_biblio( 'test' );
+
+    # test with 1 holding
```

Comment:
Use subtest here to clearly in human-readable form explain the purpose of these tests. Tests should tell a story. A funny story :)
Use Gherkin if possible.
Also when these tests fail, it is easier to find the failing test from the subtest name/output, than calculating the number of failed assertions.

Reply:
Now using subtests.

Regarding human-readable form, I'd argue it's better to add consistency. If you want to change a convention, get an approval from the community and then modify existing tests. Inconsistenct tests suck and it's hard for developers to add new code as there is no clear coding convention.

Reply:
P.S. when a test fail, it tells you the line number where the failure occured. I find it the easiest way to locate the point of failure in code.